src/site/content/en/metrics/cls/index.md

```diff
@@ -307,8 +307,14 @@ Ideally, pages would not have any layout shifts, so a CLS of `0` is the goal.
 Realistically, in some cases it's impossible to avoid 100% of layout shifts, so
 minimizing what you can is best.
 
-The Chrome Use Experience Report categorizes any page with a CLS less than 5 to
-be considered "small" (i.e. good).
+CrUX categorizes any page with a CLS less than `0.05` to be considered "small"
```

Comment:
What do we actually mean by this? We don't assign labels in CrUX.. =/

Reply:
I guess when I said CrUX I should have said the CrUX Dashboard, which does assign labels. My understanding was it was going to use the <5 bucket for "small", though now I can't find where I originally saw that. @rviscomi any ideas? If we haven't assigned a label yet, I can remove this (though I think the note is still helpful to explain that CrUX buckets are percentages).

Reply:
Ah. If memory serves, <5 is because that bucket covers the 0-5 range and ideally CLS is 0 — that's definitely a good place to be. That said, given that this is not our official guidance for CLS thresholds, I would probably suggest we drop this part, but keep the rest.

Reply:
Sure. I removed the section for now. We can re-add it once we've finalized our recommendations.